glowing_waffles/notebooks/acp.py

```diff
@@ -0,0 +1,107 @@
+class ObserveACP:
+    
+    def __init__(self, name, date):
+        
+        from astropy.coordinates import SkyCoord
+        from astropy.time import Time
+        import datetime
+    
+        self.name = name
+        self.filters = []
+        self.binning = 1
+        
+        self.ra = SkyCoord.from_name(name).ra.deg
+        self.dec = SkyCoord.from_name(name).dec.deg
+        
+        #self.date = str(datetime.datetime.today()).split()[0]
+        self.date = date
+        
+        
+    def wait_until(self, time):
+        self.startTime = time
+        
+        
+    def add_filter(self, filt):
+        
+        if filt in self.filters:
+            print("Filter", filt, "is already in list!")
```

Comment:
Should this also return at this point? Or should it be possible to specify the same filter more than once?

Reply:
I understand that functions are supposed to return, but what does a return do here?

Reply:
does the fact that I'm using if and elif emulate a return statement?

Reply:
I guess in this case -- since all of the rest of the code is `elif`s -- a return doesn't actually add anything.